.github/workflows/ios.yml

```
@@ -110,6 +110,38 @@ jobs:
         path: UnitTestResults.xcresult
         retention-days: 7
 
+    - name: Extract and Check Coverage
+      id: coverage
+      run: |
```

actionlint (1.7.10)

shellcheck reported issue in this script: SC2086:info:20:30: Double quote to prevent globbing and word splitting

(shellcheck)